sensirion_ess.cpp

```diff
@@ -172,13 +172,21 @@ int SensirionESS::measureIAQ()
 
     // SGPC3 only has TVOC; SGP30 sends [eco2, tvoc]
     if (mProductType == PRODUCT_TYPE_SGP30) {
-      mECO2 = (uint16_t)(mDataBuf[0] << 8) | mDataBuf[1];
-      if (crc8(mDataBuf+3, 2) != mDataBuf[5]) {
+      if (crc8(mDataBuf, 2) != mDataBuf[2]) {
```

Comment:
Isn't this checked on [line 168](https://github.com/Sensirion/arduino-ess/pull/16/commits/1636f47d5151969ab3cdd332698ee7357f76eb27#diff-1e316dcc8451697129a44fdbbf9c5d5aL168) above already?

Reply:
oops, yes. I think the only thing to do would be to flip the CRC check and the assignment to mECO2. Will resubmit